.settings/org.eclipse.buildship.core.prefs

```diff
@@ -0,0 +1,13 @@
+arguments=
```

Comment:
I think we shouldn't add this kind of setting files to the team repo. Can you leave this file out or put it in the .gitignore?